.eslintrc.json

```
@@ -157,6 +157,37 @@
       "rules": {
         "filenames-simple/naming-convention": "off"
       }
+    },
+    {
+      "files": ["**/api/src/**/*.ts"],
+      "plugins": ["eslint-plugin-jsdoc"],
+      "extends": ["plugin:jsdoc/recommended"],
+      "rules": {
+        "jsdoc/require-jsdoc": [
+          "warn",
```

```suggestion
          "error",
```
Since we promote `warn` to `error`, we might as well just use `error`.